src/base-command.ts

```diff
@@ -18,6 +28,24 @@ export default abstract class extends Command {
   async init() {
     if (!this.app) {
       this.app = await AppService.create(this.config.configDir);
+
+      if (this.auth_required() && !process.env.ARC_DISABLE_AUTH_CHECK) {
```

Comment:
Remove the ARC_DISABLE_AUTH_CHECK env. If there are failing tests we can mock the CredentialManager.
```ts
const credential_spy = sinon.fake.returns('token');
sinon.replace(CredentialManager.prototype, 'get', credential_spy);
```